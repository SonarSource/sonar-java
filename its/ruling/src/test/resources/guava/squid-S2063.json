{
'com.google.guava:guava:src/com/google/common/collect/Ordering.java':[
305,
],
}
