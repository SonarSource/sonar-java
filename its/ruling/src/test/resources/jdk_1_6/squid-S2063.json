{
'jdk_1_6:java/awt/datatransfer/DataFlavor.java':[
669,
],
'jdk_1_6:java/net/CookieManager.java':[
400,
],
'jdk_1_6:java/text/RuleBasedCollator.java':[
222,
],
}
