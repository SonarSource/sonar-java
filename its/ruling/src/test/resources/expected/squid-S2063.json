{
'project:com/sun/corba/se/impl/io/ObjectStreamClass.java':[
1682,
1696,
1711,
1729,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClassUtil_1_3.java':[
315,
329,
430,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClass_1_3_1.java':[
1184,
1198,
1216,
],
'project:com/sun/jmx/mbeanserver/MBeanAnalyzer.java':[
202,
],
'project:com/sun/mirror/util/SourceOrderDeclScanner.java':[
57,
],
'project:java/awt/datatransfer/DataFlavor.java':[
686,
],
'project:java/net/CookieManager.java':[
428,
],
'project:java/text/RuleBasedCollator.java':[
246,
],
'project:javax/swing/DefaultFocusManager.java':[
159,
],
'project:javax/swing/table/TableRowSorter.java':[
302,
],
}
